torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -143,20 +143,23 @@ def eq(value, other):
     dist.Backend.NCCL,
     dist.Backend.GLOO,
     dist.Backend.MPI,
+    # TODO(ucc): dist.Backend.UCC,
 ]
 
 # Allowlist of distributed backends where profiling is supported with use_cuda=True
 CUDA_PROFILING_SUPPORTED_BACKENDS = [
     dist.Backend.GLOO,
     dist.Backend.MPI,
     dist.Backend.NCCL,
+    # TODO(ucc): dist.Backend.UCC,
 ]
 
 # Allowlist of distributed backends where profiling is supported for p2p ops
 SEND_RECV_PROFILING_SUPPORTED_BACKENDS = [
     dist.Backend.MPI,
     dist.Backend.GLOO,
     dist.Backend.NCCL,
+    # TODO(ucc): dist.Backend.UCC,
```

Comment:
These failures are fixed by @Fuzzkatt in https://github.com/pytorch/pytorch/pull/84368